### PR TITLE
Add dask-jobqueue

### PR DESCRIPTION
**Pull request by jhamman**

### CODE REVIEW

File: recipes/dask-jobqueue/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  home: https://github.com/dask/dask-jobqueue
+  license: BSD 3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
+  description: |
+    Easy deployment of Dask Distributed on job queuing systems such as PBS,
+    Slurm, or SGE.
+  doc_url: http://simplejson.readthedocs.io/
+  dev_url: https://github.com/simplejson/simplejson
```
Comment (CJ-Wright) [inline recipes/dask-jobqueue/meta.yaml]: This needs to be replaced with the actual url

Comment (jhamman) [reply recipes/dask-jobqueue/meta.yaml]: thanks for the catch!

**Comment by jakirkham** [comment]
Failing with the following during the import tests.

```python
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/dask-jobqueue_1526069152115/test_tmp/run_test.py", line 2, in <module>
    import dask_jobqueue
  File "/home/conda/feedstock_root/build_artifacts/dask-jobqueue_1526069152115/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/site-packages/dask_jobqueue/__init__.py", line 2, in <module>
    from .core import JobQueueCluster
  File "/home/conda/feedstock_root/build_artifacts/dask-jobqueue_1526069152115/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.6/site-packages/dask_jobqueue/core.py", line 9, in <module>
    import docrep
ModuleNotFoundError: No module named 'docrep'
```

Should `docrep` be a `run` dependency.

**Comment by jhamman** [comment]
Thanks @jakirkham. One of these days, I'll get a recipe right first try!

**Comment by CJ-Wright** [comment]
LGTM

**Comment by jhamman** [comment]
Thanks @CJ-Wright!

**Comment by CJ-Wright** [comment]
@mrocklin @lesteve @guillaumeeb I am very sorry I forgot to ask all of you if you were ok with being maintainers. If you don't want to be a maintainer please issue a PR against the feedstock removing yourself from the list of `recipe-maintainers`. Once that PR is merged you will be removed from the team.

**Comment by jakirkham** [comment]
FYI we are having conversion problems as noted in issue ( https://github.com/conda-forge/staged-recipes/issues/5846 ).

As a nice benefit of this, it is possible to update your maintainer status for this recipe at `staged-recipes` before it is converted.

**Comment by mrocklin** [comment]
Happy to be listed as a maintainer

**Comment by mrocklin** [comment]
Thank you for creating the package!

**Pull request merged**